cabal-helper.cabal

```diff
@@ -1,5 +1,5 @@
 name:                cabal-helper
-version:             0.8.0.2
+version:             0.8.0.3
```

Comment:
I think we ought to at least do a minor version bump for new Cabal version support. I'd actually prefer to bump the version myself since its part of the release process if you don't mind.